truss/templates/tgi/tgi.Dockerfile.jinja

```diff
@@ -28,10 +23,26 @@ RUN python3 /cache_warmer.py {{file}} {{repo}} {% if hf_dir.revision != None %}{
         {% endfor %}
 {%- endif %}
 
+
+FROM ghcr.io/huggingface/text-generation-inference:0.9.4 as tgi
+EXPOSE 8080
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        nginx supervisor curl && \
+        rm -rf /var/lib/apt/lists/*
+
+
 COPY ./proxy.conf /etc/nginx/conf.d/proxy.conf
 
 RUN mkdir -p /var/log/supervisor
 COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 
 ENV SERVER_START_CMD /usr/bin/supervisord
+
+{%- if hf_cache %}
+# TODO(varun): split copy into multiple layers
```

Comment:
is it possible to explore this in the PR? The issue is that image layers are pulled in parallel, so the download speed difference in massive if we don't do it.